maizzle/src/templates/newsletterVerify.html

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Notarise: Please confirm your email"
+title: "Notarise.gov.sg: Please confirm your email address"
```

Comment:
keep the title same with other template like `Notarise:`

Reply:
Hmm ok let me clarify with the UX

Reply:
i don't think it is use in somewhere. it's just template title for maizzle right ?

Reply:
yup, just for the news letter emails